python/matasano/set1/c2.py

```diff
@@ -1,3 +1,104 @@
+<<<<<<< HEAD
```

Comment:
Hey Greg,

You need to resolve the merge properly. I've written a section about resolving merge conflicts on the tutorial (soon to be merged in).